test/s1/appframework/appframework_test.go

```diff
@@ -269,8 +243,7 @@ var _ = Describe("s1appfw test", func() {
 			// Upload V2 apps to S3
 			appVersion := "V2"
 			testenvInstance.Log.Info(fmt.Sprintf("Upload %s apps to S3 for Standalone and Monitoring Console", appVersion))
-			s3TestDir = "s1appfw-" + testenv.RandomDNSName(4)
-			appFileList := testenv.GetAppFileListPhase3(appListV2)
+			appFileList := testenv.GetAppFileList(appListV2, 2)
```

Comment:
Do we need `GetAppFileListPhase3` anymore? If not, can the definition (in appframework_utils.go) be removed too?

Reply:
This needs to be switched to `GetAppFileListPhase3`

Reply:
Modified